rust/src/lib.rs

```diff
@@ -3452,11 +2833,13 @@ impl Decimal {
 ///
 /// # Safety
 ///
-/// - `tor_runtime` must be non-null and point to a struct having the layout of
-///   [`TorRuntime`].
+/// - `tor_runtime` must be a non-null pointer returned by a `zcashlc_*` method with
+///   return type `*mut TorRuntime` that has not previously been freed.
 /// - `tor_runtime` must not be passed to two FFI calls at the same time.
 #[no_mangle]
-pub unsafe extern "C" fn zcashlc_get_exchange_rate_usd(tor_runtime: *mut TorRuntime) -> Decimal {
+pub unsafe extern "C" fn zcashlc_get_exchange_rate_usd(
+    tor_runtime: *mut TorRuntime,
+) -> ffi::Decimal {
     // SAFETY: We ensure unwind safety by:
```

Comment:
I feel like we shouldn't have comments like this that don't reflect the fact that the Swift code is not following them.
```suggestion
    // SAFETY: Callers would have to do the following for unwind safety (#194):
```